.github/workflows/linters.yml

```diff
@@ -0,0 +1,37 @@
+name: Linters
```

Comment:
- [ ] Kindly update your README with [Microverse template](https://github.com/microverseinc/readme-template) instead of `react template` and update it with your info in order to make your README more professional.